### ISSUE TITLE
yarn dev fails rust react

### ISSUE THREAD

**Issue by amiyatulu** [near/create-near-app#296]
yarn dev gives the following error for rust react app

```
13 info lifecycle new-awesome-app@0.1.0~dev:deploy:contract: Failed to exec dev:deploy:contract script
14 verbose stack Error: new-awesome-app@0.1.0 dev:deploy:contract: `near dev-deploy --wasmFile ./contract/res/status_message.wasm`
14 verbose stack Exit status 1

cp: no such file or directory: ./target/wasm32-unknown-unknown/release/status_message.wasm
```

**Comment by amiyatulu** [comment]
Worked after running this command
`rustup target add wasm32-unknown-unknown`

https://docs.near.org/docs/roles/developer/contracts/near-sdk-rs

**Comment by janedegtiareva** [comment]
Thank you for reporting! It looks like we don't have this step documented in the README. We'll make the necessary fixes.

**Comment by chadoh** [comment]
Thanks for the report, @amiyatulu ! This was implemented in #389 and will be shipped with version 1.1.0 within the next couple days
